packages/gp2-components/src/molecules/CardWithCornerBackground.tsx

```diff
@@ -0,0 +1,41 @@
+import { Card, pixels } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import { ComponentProps } from 'react';
+import { projectsImage } from '../images';
+
+type CardWithCornerBackgroundProps = ComponentProps<typeof Card>;
+
+const { rem } = pixels;
+
+const containerStyles = css({
+  display: 'grid',
+  gridTemplateRows: '52px 1fr 22px',
+  gridTemplateColumns: '58px 1fr 28px',
+  margin: '12px 0',
+});
+
+const backgroundStyle = css({
+  gridArea: '2 / 2 / 4 / 4',
+  order: -1,
```

Comment:
you can just have the background come before the content div. I think its easier to understand than changing ordering